### ISSUE TITLE
[question] is it compatible with v45?

### ISSUE THREAD

**Issue by ilkeEren27** [vasanex/ItemQuickSwitchMod#5]
The ThunderStore says the last update date of mod was 3 weeks ago, is it compatible with the new version of game that came a few days ago?

**Comment by vasanex** [comment]
Yes it should be compatible without any changes. Feel free to try it out.

That being said I am going to work on an update to integrate it into the new settings menu soon.
